examples/trinket/.cargo/config.toml

```
@@ -0,0 +1,8 @@
+[build]
+target = "../../avr-specs/avr-attiny85.json"
+
+[target.'cfg(target_arch = "avr")']
+runner = "ravedude trinket -cb 57600"
```

I don't think we'll expect a console on the trinket so I would remove `-cb 57600` here:

```suggestion
runner = "ravedude trinket"
```